nightfall-client/src/routes/finalise-withdrawal.mjs

```diff
@@ -12,11 +12,11 @@ const router = express.Router();
 router.post('/', async (req, res, next) => {
   logger.debug(`finalise-withdrawal endpoint received POST ${JSON.stringify(req.body, null, 2)}`);
   try {
-    const { rawTransaction: txDataToSign } = await finaliseWithdrawal(req.body);
+    const { rawTransaction: txDataToSign, transaction } = await finaliseWithdrawal(req.body);
```

Comment:
We don't seem to be using the `transaction` that was added here. 
Could we remove this please?

Reply:
The second transaction that its signed when requesting an instant withdrawal is the request of an instant withdraw. 
```  /**
  Enables someone with a valid withdraw transaction in flight to request instant
  withdrawal of funds (only relevant for ERC20).
  @method
  @async
  @param {string} withdrawTransactionHash - the hash of the Layer 2 transaction in question
  @param {number} fee - the amount being paid for the instant withdrawal service
  */
  async requestInstantWithdrawal(withdrawTransactionHash, fee) {
    try {
      // set the instant withdrawal fee
      const res = await axios.post(`${this.clientBaseUrl}/set-instant-withdrawal`, {
        transactionHash: withdrawTransactionHash,
      });
      return this.submitTransaction(res.data.txDataToSign, this.shieldContractAddress, fee);
    } catch {
      return null;
    }
  }```

Reply:
I thought this was the first and only transaction the user signs

Reply:
The first transaction the user signs is the withdraw request. The second is the instant withdraw request

Reply:
That makes sense. Thank you